config/packages/biblioverse_typesense.yaml

```diff
@@ -77,6 +77,18 @@ biblioverse_typesense:
             optional: true
             type: int32[]
             entity_attribute: users.favorite
+          embedding:
+            name: embedding
+            type: float[]
+            index: true
+            mapped: false
+            numDim: 768
+            embed:
+              from: ["extension", "tags", "summary"]
```

Comment:
I don't think I would put extension, maybe the title instead

Reply:
No, the extension makes it more likely to have other books when you are looking for a book or comics when looking for a comic, but that's just a little nice to have. the title however skews the result too much, especially for Mangas where you'll have similarities on "Volume 01", etc. In another future MR, I'll look into how I could balance the weight of these fields, but currently this is the best results I could manage